pkg/utils/taints.go

```diff
@@ -51,21 +52,21 @@ func InitOutOfServiceTaintSupportedFlag(config *rest.Config) error {
 		if cs == nil {
 			err = fmt.Errorf("k8s client set is nil")
 		}
-		utilTaintsLog.Error(err, "couldn't get retrieve k8s client")
+		utilTaintsLog.Error(err, "couldn't retrieve k8s client")
 		return err
 	} else if version, err := cs.Discovery().ServerVersion(); err != nil || version == nil {
 		if version == nil {
 			err = fmt.Errorf("k8s server version is nil")
 		}
-		utilTaintsLog.Error(err, "couldn't get retrieve k8s server version")
+		utilTaintsLog.Error(err, "couldn't retrieve k8s server version")
 		return err
 	} else {
 		var majorVer, minorVer int
 		if majorVer, err = strconv.Atoi(version.Major); err != nil {
 			utilTaintsLog.Error(err, "couldn't parse k8s major version", "major version", version.Major)
 			return err
 		}
-		if minorVer, err = strconv.Atoi(version.Minor); err != nil {
+		if minorVer, err = strconv.Atoi(strings.ReplaceAll(version.Minor, "+", "")); err != nil {
```

Comment:
what about putting this else block into it's own func with a unit test?

Reply:
Good idea.